lib/features/children/generosity_challenge/pages/generosity_challenge_overview.dart

```diff
@@ -169,35 +172,50 @@ class _GenerosityChallengeOverviewState
                     );
                   },
                 ),
-                const Spacer(),
                 if (widget.isDebug)
-                  ToggleButtons(
-                    borderRadius: const BorderRadius.all(Radius.circular(8)),
-                    selectedBorderColor: Colors.blue[700],
-                    selectedColor: Colors.white,
-                    fillColor: Colors.blue[200],
-                    color: Colors.blue[400],
-                    constraints: const BoxConstraints(
-                      minHeight: 40,
-                      minWidth: 80,
+                  Padding(
+                    padding: const EdgeInsets.only(top: 8),
+                    child: ToggleButtons(
+                      borderRadius: const BorderRadius.all(Radius.circular(8)),
+                      selectedBorderColor: Colors.blue[700],
+                      selectedColor: Colors.white,
+                      fillColor: Colors.blue[200],
+                      color: Colors.blue[400],
+                      constraints: const BoxConstraints(
+                        minHeight: 40,
+                        minWidth: 80,
+                      ),
+                      isSelected: [
+                        challenge.state.unlockDayTimeDifference ==
+                            UnlockDayTimeDifference.days,
+                        challenge.state.unlockDayTimeDifference ==
+                            UnlockDayTimeDifference.minutes,
+                      ],
+                      onPressed: challenge.toggleTimeDifference,
+                      children: [
+                        Text(UnlockDayTimeDifference.days.name),
+                        Text(UnlockDayTimeDifference.minutes.name),
+                      ],
                     ),
-                    isSelected: [
-                      challenge.state.unlockDayTimeDifference ==
-                          UnlockDayTimeDifference.days,
-                      challenge.state.unlockDayTimeDifference ==
-                          UnlockDayTimeDifference.minutes,
-                    ],
-                    onPressed: challenge.toggleTimeDifference,
-                    children: [
-                      Text(UnlockDayTimeDifference.days.name),
-                      Text(UnlockDayTimeDifference.minutes.name),
-                    ],
                   ),
+                const Spacer(),
               ],
             ),
           ],
         ),
       ),
+      floatingActionButton: GivtElevatedButton(
```

Comment:
I don't think it should be an floatingactionbutton, but just a button at the bottom of the page.
floatingactionbutton is only if it needs to be on top of other things